src/main/java/org/codewars/utils/TaskRunner.java

```diff
@@ -46,8 +47,18 @@ public static String runStockSummary(){
         return null;
     }
 
-    public static double runGetVolumeOfCuboid(){
-        return 0.0;
+    public void runGetVolumeOfCuboid(){
+
+        System.out.println("Run task GetVolumeOfCuboid");
```

Comment:
I'll suggest it for all: would it be better to use task names (Volume of a Cuboid) instead of method names?

Reply:
Agree. Done.